.kitchen.yml

```diff
@@ -1,19 +1,21 @@
 ---
 driver:
   name: vagrant
+  cachier: true
+  cpus: 2
 
 platforms:
-  - name: debian/jessie64
-  - name: ubuntu/xenial64
-  - name: centos/7
-  - name: centos/6
+  - name: bento/debian-8
+  - name: bento/ubuntu-16.04
+  - name: bento/centos-6
+  - name: bento/centos-7
 
 provisioner:
   name: salt_solo
-  log_level: debug
-  salt_version: '2016.3'
```

Comment:
I prefer to developing against the latest version of a software package, so it forces me to fix any broken code early on. Otherwise, you end up using an old version for a while and when you decide to update, you need to spend a fair amount of time fixing all your code.